core/src/com/unciv/ui/screens/diplomacyscreen/OffersListScroll.kt

```diff
@@ -103,11 +103,13 @@ class OffersListScroll(
                 }
 
                 val amountPerClick =
-                        if (offer.type == Gold) 50
-                        else 1
+                    when (offer.type) {
+                        Gold -> 50
+                        Treaty -> Int.MAX_VALUE
```

Comment:
Why...?

Reply:
The research agreement has a gold value according to its cost. In order to move the entire research agreement from available offers to offers we need to have this at Int.Max_VALUE.